docs/src/fit.md

```diff
@@ -72,7 +72,9 @@ fit_mle(Categorical, k, x, w)
 fit_mle(Categorical, x)        # equivalent to fit_mle(Categorical, max(x), x)
 fit_mle(Categorical, x, w)
 ```
+!!! warning
 
+     Laplace distribution currently does not support weighted `fit_mle`.
```

Comment:
```suggestion
     Laplace distribution currently does not support weighted `fit_mle`.
     
```

Reply:
Pardon, but aren't both the lines same?

Reply:
this is just adding a new line before the next header